### PR TITLE
Nftables

### PR DESCRIPTION
**Pull request by ReillyBrogan**
**Summary**
- Update or rebuild entire iptables/nftables stack
- Switch to using nftables by default instead of legacy xtables
- Update dependencies to depend on iptables
- Remove ebtables and libnetfilter_acct

Closes getsolus/packages#2132

Note that though this PR removes both ebtables and libnetfilter_acct only ebtables is added the the deprecation list. As far as I can tell libnetfilter_acct isn't in the package repo at all, it was probably committed back in 2017 but never built.

### CODE REVIEW

File: packages/i/iptables/files/0001-Stateless.patch [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+diff --git a/Makefile.am b/Makefile.am
+index 299ab46d..3f675d35 100644
+--- a/Makefile.am
++++ b/Makefile.am
+@@ -19,7 +19,7 @@ SUBDIRS         += iptables
+ EXTRA_DIST	= autogen.sh iptables-test.py xlate-test.py
+ 
+ if ENABLE_NFTABLES
+-confdir		= $(sysconfdir)
++confdir		= /usr/share/etc
```
Comment (ermo) [inline packages/i/iptables/files/0001-Stateless.patch:21]: Is this path correct?

**Review by ermo — DISMISSED** [review]
Patch path looks wonky?

File: packages/n/nftables/package.yml [hunk 2]
```diff
@@ -14,23 +16,34 @@ builddeps  :
...
-    %configure \
-        --disable-static \
-        --with-json \
-        --disable-debug \
-        --with-cli=readline \
-        --with-python-bin=/usr/bin/python3
+    %patch -p1 -i $pkgfiles/0001-Stateless.patch
+    %reconfigure \
+                 --disable-debug \
+                 --disable-python \
```
Comment (ermo) [inline packages/n/nftables/package.yml:32]: You're the expert here, but why are you disabling python, yet shipping python-nftables...? Is it to do with disabling being able to programmatically interface with nftables via a python C module?

**Review by ermo — COMMENTED** [review]
LOL at this last being touched by serebit xD

File: packages/n/nftables/package.yml (hunk 2 above)
Comment (ReillyBrogan) [reply packages/n/nftables/package.yml:32]: We're disabling python as part of the main build, but building it later with the `python3 -m build --wheel` command.

File: packages/i/iptables/files/0001-Stateless.patch (hunk 1 above)
Comment (ReillyBrogan) [reply packages/i/iptables/files/0001-Stateless.patch]: Fixed

**Review by silkeh — APPROVED** [review]
LGTM! Should we follow this up with some instructions on how users should do nftables modifications, or are users expected not to do this?

**Comment by ReillyBrogan** [comment]
> LGTM! Should we follow this up with some instructions on how users should do nftables modifications, or are users expected not to do this?

I believe that users with existing iptables rules won't need to make any changes, iptables will simply apply them using the nftables kernel interfaces instead. Same with docker/lxd/libvirtd/firewalld, they use the iptables binaries directly and will just shift to using nftables instead. We do need to have a note on the sync thread for it for people doing things more manually.

**Pull request merged**